app/src/main/java/com/example/encoratask/model/Character.kt

```diff
@@ -0,0 +1,46 @@
+package com.example.encoratask.model
+
+import org.json.JSONObject
+import java.io.Serializable
+
+/**
+ * Created by Erick Sanchez
+ * Revision 1 - 09/06/21
+ */
+class Character:Serializable {
+
+    var id:Int = 0
+    var name:String = ""
+    var status = ""
+    var species = ""
+    var gender = ""
+    var image:String = ""
+
+    constructor(obj:JSONObject) {
+
+        this.id = obj.optInt("id")
```

Comment:
Names of your JSON attributes might change in time. You should be adding this names in a proper place that you can change fast. I would recommend the companion block to add const values with the names at least

Reply:
soudns good, but in many cases you always gonna use specific API, and if the API change, there is a bigger bussines problem